src/frontend/templatehelpers/templatetags/custom_time_formatting.py

```diff
@@ -0,0 +1,18 @@
+from django import template
+
+register = template.Library()
+
+# Peformance hack for converting times to a fixed ISO-like format noticeably faster
```

Comment:
Nice to know. 

Do you know the performance impact of just this ?

Reply:
Fairly small, like 10ms or something for 500 networks. But it's a pretty simple change, I think, and doesn't break any abstraction boundaries.